cupboard_app/queries.py

```diff
@@ -196,14 +249,16 @@ def get_user(username: str, id: int = None) -> User | None:
     return result
 
 
-def create_list_ingredient(ingredient: str, amount: int | float, unit: str) -> dict:
+def create_list_ingredient(ingredient: str, amount: int | float,
+                           unit: str, user_id: int = None) -> dict:
```

Comment:
For moving the parameters to the next line due to the length, do this instead which is the standard practice:
```
def create_list_ingredient(
    ingredient: str,
    amount: int | float,
    ...
) -> dict:
```